### ISSUE TITLE
Rectangle is not converted to polygon

### ISSUE THREAD

**Issue by mzur** [biigle/reports#130]
While COCO only supports segmentations and bboxes, lines and rectangles are also exported. In the upcoming COCO import, these could be imported, too, but there is an abiguous case. In the import lines and rectangles can be identified by their first coordinate not matching the last coordinate. Rectangles have 4 coordinates and lines are all the rest. But this way a line with 4 coordinates would be incorrectly identified as a rectangle. So in the COCO export we should just convert rectangles to polygons to solve this issue for the import.

Basically we only have to add a special case for rectangles [here](https://github.com/biigle/reports/blob/7333bcea14bd9e0b40148f9fb02f681e523eacec/src/resources/scripts/to_coco.py#L58-L105) and append the first coordinate to their points array.
